docs/self-hosted/oel/oauth2/token-prefix.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: token-prefix
+title: Custom OAuth2 Token Prefix
+sidebar_label: Custom Token Prefix
+---
+
+# Configurable Token Prefix for OAuth2 Tokens
+
+This documentation explains how to configure the token prefix for OAuth2 access tokens, refresh tokens, and authorization codes.
+By default, all tokens use the `ory_%s_` template, where `%s` is replaced with the token type.
+
+Customizing the token prefix is available exclusively to Ory Enterprise License customers.
+
+## Default Token Prefix
+
+By default, OAuth2 tokens use the following template: `ory_%s_`. The `%s` is a placeholder that will be replaced by the specific
+token type:
+
+- `rt` for refresh tokens
+- `at` for access tokens
+- `ac` for authorization codes
+
+### Examples:
+
+- Refresh Token: `ory_rt_...`
+- Access Token: `ory_at_...`
+- Authorization Code: `ory_ac_...`
+
+## Customizing the Token Prefix
+
+With the `oauth2.token_prefix` configuration key, you can customize the prefix for your OAuth2 tokens. This key is only available
+to Ory Enterprise License customers.
+
+### How to Configure
+
+1. **Locate the Configuration File**: Open your Ory configuration file.
+2. **Set the `oauth2.token_prefix` Key**: Add or modify the `oauth2.token_prefix` key in your configuration file.
+
+### Example Configuration
+
+```yaml
+oauth2:
+  token_prefix: "acme_token_%s_"
+```
+
+In this example, tokens will use the `acme_token_%s_` template. The `%s` placeholder for the token type must always be included in
+the template.
+
+### Resulting Tokens
+
+- Refresh Token: `acme_token_rt_...`
+- Access Token: `acme_token_at_...`
+- Authorization Code: `acme_token_ac_...`
+
+## Important Notes
```

Comment:
maybe rather use :::info here and put it at the top of the document, then we also dont need to mention 4 times that this is only available for OEL in this short document.